services/drupal/composer.patches.json

```diff
@@ -77,9 +77,9 @@
       "Do not reimport existing entities": "https://www.drupal.org/files/issues/2020-11-06/default_content-existing-entities-2698425-1-x.patch"
     },
     "drupal/content_moderation_notifications": {
-        "Group Entity Support": "https://www.drupal.org/files/issues/2020-02-17/content_moderation_notifications-3015275-35.patch",
-        "Add option to email Revision Author": "patches/content-moderation-notifications-3133940-2-rolled-to-apply-with-3015275.patch",
-        "Flag module support": "patches/content-moderation-notifications-3316633-flag-support-rolled-to-apply-with-3015275-and-3133940.patch"
+      "Group Entity support": "patches/content_moderation_notifications-with-issues-3015276-comment-14511793-patch-applied.patch",
+      "Add option to email Revision Author": "patches/content_moderation_notifications_issues-3133940-comment-14693459-patch-over-issues-3015276-comment-14511793-patch.patch",
```

Comment:
Let's clean up the naming of these patches.  I'd like them to be named content_moderation_notifications-[issue ID of the issue the patch is meant to fix]-[comment # (not ID, just the number of the comment on the issue) that contains the patch that this patch was based on]-rolled-to-apply-with-[issue IDs of each previously-applied-patch].patch.  

For example: `content-moderation-notifications-3133940-2-rolled-to-apply-with-3015275.patch`